apps/docs/src/lib/components/TopBar/TopBar.svelte

```diff
@@ -180,15 +205,19 @@
 				style={`text-decoration: none`}
 				on:click={() => addSearch(matchingSearch)}
 			>
-				<Paper class="searchTerm" withBorder>
-					<p class="searchItemTitle">{matchingSearch.title}</p>
-					{#if matchingSearch.section}
-						<p class="searchItemDescription">{matchingSearch.section}</p>
-					{/if}
-				</Paper>
+				<SearchItem search={matchingSearch} />
 			</a>
 		{/each}
 	{:else}
 		<p class="noMatches">No matches</p>
 	{/if}
+	<Divider  />
```

Comment:
I like this 👍 In the future, when we have an Accordion component, we could collapse each section, so that it does not take so much space

Reply:
Good idea 👍